src/vaults/src/entities/strategy.ts

```diff
@@ -33,7 +33,6 @@ export function loadStrategyV1_5(address: Address): Strategy {
   let id = address.toHexString();
   let strategy = Strategy.load(id) as Strategy;
   if (strategy == null) {
-    BadgerStrategy.create(address);
```

Comment:
strategy v1.5 has no meaningful events to listen to - as such we don't need a handler and further more should NOT create a data source